package.json

```diff
@@ -73,7 +73,7 @@
     "./packages/*"
   ],
   "engines": {
-    "node": ">=12.0.0",
-    "npm": ">=7.0.0 < 10.0.0"
+    "node": ">=16.0.0",
```

Comment:
I think we dont have an explicit top range in hopes that it "just works" in future releases.

Reply:
Oh also in the README `**[Node.js v16](https://nodejs.org/)** is recommended. We also support v18 and v20. Other versions may work but are not frequently tested.`

Reply:
Yeah that is what we test and support but people can try higher.  Lower will be an issue because there will be changes made to dependencies that wont work less than 16.

Reply:
Since newer versions of Node are mostly backwards compatible, then I would say it's fine to exclude. Just curious :)